pkg/openshift/template.go

```diff
@@ -0,0 +1,127 @@
+package openshift
```

Comment:
Should we separate between "pkg/openshift" the Operator stuff and the "pkg/openshift" 'Other stuff'?

Reply:
maybe pkg/openshift/client ?

Reply:
or move istio|k8s|openshift "operator stuff" into a common parent package..

Reply:
I did a small shuffling around. I opened #150 issue to clean up the client stuff.